server/models/Column.js

```diff
@@ -0,0 +1,14 @@
+const mongoose = require("mongoose");
+
+const columnSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+    unique: false
+  },
+  cards: [{
+    type: String,
+  }]
```

Comment:
Same here with the ObjectId type with a ref to cards

Reply:
Got it.